docs/ethical-advertising.rst

```diff
@@ -0,0 +1,123 @@
+There is one proven business model to support this kind of site:
+Advertising.
+
+We talk a bit :ref:`below <ethical-info>` about our philosophy on advertising.
+
+If you're a marketer,
+we have :ref:`information <ethical-buy-ads>` on how to get in touch with us about running ads.
+
+.. _ethical-info:
+
+Our worldview
+-------------
+
+We're doing advertising properly:
+
+* We don't track users
+* We don't sell your data
+* We host everything ourselves, no third-party scripts
+
+What we're doing is newspaper advertising,
+on the internet.
+For a hundred years,
+newspapers put an ad on the page,
+some folks would see it,
+and advertisers would pay for this.
+This is our model.
+
+So much ad tech has been built to track users.
+You follow them across them web,
+from site to site,
+showing them the same ad.
+Then retailers sell your purchase data to try and attribute sales to advertising.
+Now there is an industry in doing `fake ad clicks`_ and other scams,
+which leads the ad industry to track you even more intrusively to know more about you.
+The current advertising industry is a vicious downward spiral.
+**We opt out.**
+
+As developers,
+we understand the `massive downsides`_ of the current advertising industry.
+From malware,
+slow site performance,
+and huge databases of your personal data sold to the highest bidder.
+**We opt out.**
+
+The trend in advertising is to have larger and larger ads.
+They should run before your content,
+they should take over the page,
+the bigger, weirder, or flashier the better.
+**We opt out.**
+
+We don't store personal information about you.
+We keep track of views and clicks.
+We don't build a profile of your personality to sell ads against.
+We only show high quality ads from companies that are of interest to developers.
+
+We are running a single,
+small,
+unobtrusive ad on documentation pages.
+The products should be interesting to you.
+They won't flash or move.
+
+We run the ads we want to have on our product,
+in a way that makes us feel good.
+
+.. _fake ad clicks: https://en.wikipedia.org/wiki/Click_fraud
+
+Join us
+-------
+
+We're building the advertising product that we wished existed.
+We hope that others will join us in this mission:
+
+* **If you're a developer**,
+  talk to your marketing people about using products that respect user privacy.
+* **If you're a marketer**,
+  vote with your dollars and support us in building the internet we want to exist.
+
+.. _massive downsides: http://idlewords.com/talks/what_happens_next_will_amaze_you.htm
+
+.. _ethical-buy-ads:
+
+Buy ads
+-------
+
+If you like our vision,
+we're love to work with you about getting your ad in front of the Read the Docs audience.
+You can fill out the basic form below,
+and we'll get in touch.
+
+**We only work with people who sell products that would be of interest to our audience of programmers and users of open source.**
+
+Advertising Interest Form
+~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
It makes it look nicer in the output -- otherwise the form is just floating off in space.